admin/controllers/results.js

```diff
@@ -50,6 +52,8 @@ controller.getViewResultsPage = async (req, res, next) => {
     })
   }
   const pupilData = resultPresenter.getResultsViewData(pupils)
+  const nationalScore = resultPresenter.getScoreWithOneDecimalPlace(checkWindow.score)
```

Comment:
I don't much like the implementation-specific method name.  What if the business decides to change the business rules?  There are really two methods here: `getScore` and `formatScore`.

Reply:
@jon-shipley I do not see any reason to introduce `getScore` as the services already fetch the scores but I do agree and have introduced `formatScore` to reflect a more generic purpose for the presenter method